projects/plugins/jetpack/tests/php/modules/shortcodes/test-class.recipe.php

```diff
@@ -185,12 +179,12 @@ function_exists( 'wp_lazy_loading_enabled' )
 			&& wp_lazy_loading_enabled( 'img', 'wp_get_attachment_image' )
 		) {
 			$this->assertContains(
-				'<img src="http://example.org/wp-content/uploads/example.jpg" class="jetpack-recipe-image u-photo photo" alt="" loading="lazy" itemprop="image" />',
+				'src="' . $url . '" class="jetpack-recipe-image u-photo photo" alt="" loading="lazy" itemprop="image" />',
 				$shortcode_content
 			);
 		} else {
 			$this->assertContains(
-				'<img src="http://example.org/wp-content/uploads/example.jpg" class="jetpack-recipe-image u-photo photo" alt="" itemprop="image" />',
+				'src="' . $url . '" class="jetpack-recipe-image u-photo photo" alt="" itemprop="image" />',
```

Comment:
Cutting off the initial `<img` since now, since it is a real image, it includes size information, e.g. `<img width="123"`. We could be more specific, but I would think it is okay to have the rest of the string.